trees/c-or-cpp/count-and-sum-of-nodes-in-binary-tree.cpp

```diff
@@ -0,0 +1,77 @@
+#include <iostream>
```

Comment:
please add a user friendly description about the problem

Reply:
ok sure i am doing it